rskj-core/src/test/java/co/rsk/peg/federation/VoteFederationChangeTest.java

```diff
@@ -459,28 +495,35 @@ void voteCommitFederation_postRSKIP419_whenPendingFederationIsSet_shouldPerformC
         assertNoHandoverToNewFederation();
     }
 
-    private void assertIsTheExpectedFederation(Federation federation, long expectedCreationTimeValue, Instant expectedCreationTime) {
-        assertEquals(RSK_EXECUTION_BLOCK_TIMESTAMP, expectedCreationTimeValue);
-        assertEquals(RSK_EXECUTION_BLOCK_NUMBER, federation.getCreationBlockNumber());
 
-        Federation federationBuiltFromPendingFederation = pendingFederationToBe.buildFederation(
-            expectedCreationTime,
-            RSK_EXECUTION_BLOCK_NUMBER,
-            federationMainnetConstants,
-            activations
-        );
-        assertEquals(federationBuiltFromPendingFederation, federation);
+    @Test
+    void commitProposedP2shErpFederation_shouldPerformCommitProposedFederationActions() {
+        // arrange
+        storageProvider.setNewFederation(activeFederation);
+
+        List<UTXO> activeFederationUTXOs = BitcoinTestUtils.createUTXOs(10, activeFederation.getAddress());
+        bridgeStorageAccessor.saveToRepository(NEW_FEDERATION_BTC_UTXOS_KEY.getKey(), activeFederationUTXOs, BridgeSerializationUtils::serializeUTXOList);
+
+        Federation proposedFederation = P2shErpFederationBuilder.builder().build();
+        storageProvider.setProposedFederation(proposedFederation);
+
+        // act
+        federationSupport.commitProposedFederation();
+
+        // assert
+        assertFalse(federationSupport.getProposedFederation().isPresent());
+        assertHandoverToNewFederation(activeFederationUTXOs, proposedFederation);
     }
 
     @Test
-    void commitProposedFederation_shouldPerformCommitProposedFederationActions() {
+    void commitProposedP2shP2wshErpFederation_shouldPerformCommitProposedFederationActions() {
```

Comment:
```suggestion
    void commitProposedFederation_shouldPerformCommitProposedFederationActions() {
```
also the method name doesnt change

Reply:
done